packages/stryker/test/unit/reporters/ScoreReporter/StrykerBadgeClientSpec.ts

```diff
@@ -0,0 +1,78 @@
+import StrykerBadgeClient from '../../../../src/reporters/ScoreReporter/StrykerBadgeClient';
+import { StrykerBadgeReport } from '../../../../src/reporters/ScoreReporter/StrykerBadgeClient';
+import { HttpClient } from 'typed-rest-client/HttpClient';
+import { Mock, mock } from '../../../helpers/producers';
+import { expect } from 'chai';
+import { Logger } from 'log4js';
+import currentLogMock from '../../../helpers/log4jsMock';
+
+describe('StrykerBadgeClient', () => {
+
+  let sut: StrykerBadgeClient;
+  let badgeClient: Mock<HttpClient>;
+  let log: Mock<Logger>;
+
+  const badgeReport: StrykerBadgeReport = {
+    api_key: '1',
```

Comment:
We shouldn't use underscores in property names. That is not the typescript/javascript way.

Reply:
Done